src/command_modules/azure-cli-network/tests/test_network_commands.py

```diff
@@ -15,7 +15,7 @@
 from azure.cli.core.commands.client_factory import get_subscription_id
 from azure.cli.core.test_utils.vcr_test_base import (VCRTestBase, ResourceGroupVCRTestBase, JMESPathCheck,
                                            NoneCheck, MOCKED_SUBSCRIPTION_ID)
-
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, StorageAccountPreparer
```

Comment:
They were used but I had to rewrite the test to the old test base because it didn't work for VM create and I was trying to get it wrapped up (which didn't happen).  TL/DR I'll remove the references :)